build.py

```diff
@@ -249,6 +249,8 @@ def unpack(self):
                 zf.extractall(path=destdir)
         # .. and set the builder object to point to the file
         self.builder.ninja = destfile

```

Comment:
I look better and the ninjia.exe is never called directly with the full path so using only the path should be ok.